Scribe/Colors/ColorVariables.swift

```diff
@@ -23,30 +23,30 @@ import UIKit
 var scribeKeyIcon = UIImage(named: "scribeKeyIcon")
 
 // Initialize all colors.
-var keyColor = UIColor(.key)
-var keyCharColor = UIColor(.keyChar)
-var specialKeyColor = UIColor(.keySpecial)
-var keyPressedColor = UIColor(.keyPressed)
+var keyColor = UIColor(ScribeColor.key)
```

Comment:
Thanks for also going through and changing color assignments! I agree that it makes sense for us to be explicit with the source :)